libcaf_test/caf/test/runner.cpp

```diff
@@ -68,6 +68,13 @@ int runner::run(int argc, char** argv) {
     return EXIT_SUCCESS;
   }
   default_reporter->start();
+  default_reporter->disable_colored(get_or(cfg_, "no-colors", false));
```

Comment:
Reading this line here, I can see why you chose to give the function an argument. But then I'd go with something like this:

```c++
  default_reporter->colored_output(!get_or(cfg_, "no-colors", false));
```

If you prefer the `disable` style, I think this would make more sense:

```c++
  if (get_or(cfg_, "no-colors", false)))
    default_reporter->disable_colors();
```